account_fiscal_position_partner_type/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account Fiscal Position Partner Type",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
```

Comment:
The version number should not be changed in the imp, it should be changed in the migration commit